[2025-10-04T23:28:21.849+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingest_data_dag.ingest_users_data manual__2025-10-04T23:17:44.635799+00:00 [queued]>
[2025-10-04T23:28:21.856+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingest_data_dag.ingest_users_data manual__2025-10-04T23:17:44.635799+00:00 [queued]>
[2025-10-04T23:28:21.857+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 4
[2025-10-04T23:28:21.869+0000] {taskinstance.py:2191} INFO - Executing <Task(S3ToSqlOperator): ingest_users_data> on 2025-10-04 23:17:44.635799+00:00
[2025-10-04T23:28:21.874+0000] {standard_task_runner.py:60} INFO - Started process 2453 to run task
[2025-10-04T23:28:21.878+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ingest_data_dag', 'ingest_users_data', 'manual__2025-10-04T23:17:44.635799+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/dag_ingest_data.py', '--cfg-path', '/tmp/tmp3v56dh3k']
[2025-10-04T23:28:21.881+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask ingest_users_data
[2025-10-04T23:28:21.937+0000] {task_command.py:423} INFO - Running <TaskInstance: ingest_data_dag.ingest_users_data manual__2025-10-04T23:17:44.635799+00:00 [running]> on host ff034d7c1964
[2025-10-04T23:28:22.040+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingest_data_dag' AIRFLOW_CTX_TASK_ID='ingest_users_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-04T23:17:44.635799+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-04T23:17:44.635799+00:00'
[2025-10-04T23:28:22.041+0000] {s3_to_sql.py:105} INFO - Loading users.csv to SQL table "staging"."users"...
[2025-10-04T23:28:22.051+0000] {base.py:83} INFO - Using connection ID 'minio_s3' for task execution.
[2025-10-04T23:28:22.052+0000] {connection_wrapper.py:378} INFO - AWS Connection (conn_id='minio_s3', conn_type='aws') credentials retrieved from login and password.
[2025-10-04T23:28:22.328+0000] {base.py:83} INFO - Using connection ID 'postgres_warehouse' for task execution.
[2025-10-04T23:28:22.339+0000] {base.py:83} INFO - Using connection ID 'postgres_warehouse' for task execution.
[2025-10-04T23:28:22.510+0000] {sql.py:544} INFO - Loaded 1000 rows into "staging"."users" so far
[2025-10-04T23:28:22.510+0000] {sql.py:547} INFO - Done loading. Loaded a total of 1000 rows into "staging"."users"
[2025-10-04T23:28:22.519+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ingest_data_dag, task_id=ingest_users_data, execution_date=20251004T231744, start_date=20251004T232821, end_date=20251004T232822
[2025-10-04T23:28:22.573+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-04T23:28:22.591+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
